xarray/core/variable.py

```diff
@@ -977,7 +974,7 @@ def chunk(self, chunks=None, name=None, lock=False):
             chunks = self.chunks or self.shape
 
         data = self._data
-        if isinstance(data, da.Array):
+        if is_duck_dask_array(data):
             data = data.rechunk(chunks)
```

Comment:
is `pint` forwarding `.rechunk` to `dask.array`? Similarly below in `_shift_one_dim`

Reply:
Yes, but not explicitly. Pint checks the wrapped dask array for the `rechunk` method and finds it, which (seems to) correctly call `dask.array.Array.rechunk`.

Reply:
:+1: